### DISCUSSION TITLE
Modal.confirm 弹出确认框， destroy 后， Modal 的 dom 还在， 导致整个页面操作不了

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#52032] [General]
最近有客户说整个界面操作不了， 一直没有重现， 今天远程看了一下， 看到最顶级有一个 Modal 的 dom, 没有隐藏， 但是内容都是不可见的， 整个界面操作不了， 隐藏这个 div 就可以了， 经过检查， 发现是一个

Modal.confirm 的弹窗：

const modal = Modal.confirm({
      title: '确定提交当前流程？',
      content: '确定提交当前流程？',
      onOk: () => {
          modal.destroy();
          // 提交操作
      },
      onCancel: () => {
          modal.destroy();
      }
  });

点确定后， 内容都透明了， 但是 DOM 没有移除， 导致整个界面都操作不了了。

这个问题很难重现， 目前就出现在一台电脑上， 电脑是华为 MATEBOOK 2024 X PRO,  不知道怎么解决这个问题

**Comment by yoyo837** [comment]
上复现
